js/fuzzysearch.js

```diff
@@ -50,6 +50,20 @@ $(function() {
         trigger_fuzzy();
     });
 
+    // when the shortcut for entity form is called
+    hotkeys('ctrl+alt+e, option+command+e', function(e) {
+        open_entity_selection(e);
+    });
+
+    async function open_entity_selection(e) {
+        e.stopPropagation();
+        e.preventDefault();
+        $('.avatar').trigger("click");
+        await new Promise(r => setTimeout(r, 100));
+        $('.dropdown-list-entity').dropdown('toggle');
+        $('input[name=entsearchtext]').filter(":visible")[0].focus();
+    }
+
```

Comment:
This code should not be here, as it is not related to fuzzy search. Maybe it should be moved into `templates/layout/parts/profile_selector.html.twig`, to be placed at the same place as the HTML code it controls..